shared/src/authorization/authorizationClientService.js

```diff
@@ -117,6 +117,12 @@ const AUTHORIZATION_MAP = {
     ROLE_PERMISSIONS.UPLOAD_DOCUMENT,
     ROLE_PERMISSIONS.VIEW_DOCUMENTS,
   ],
+  irsSuperuser: [
+    ROLE_PERMISSIONS.ADVANCED_SEARCH,
+    ROLE_PERMISSIONS.GET_CASE,
+    ROLE_PERMISSIONS.VIEW_DOCUMENTS,
+    ROLE_PERMISSIONS.VIEW_SEALED_CASE,
+  ],
```

Comment:
Each permission used in an interactor/use-case _should_ have an associated unit test with its associated authorization check, and this is reflected by SonarCloud's report for test coverage on the specific authorization mapping. Each lambda function calls an interactor, and _most_ (except those on the public API stack) of these interactors called by a lambda have an authorization check associated with it. Each permission can be checked by searching for its associated `ROLE_PERMISSIONS.<PERMISSION_NAME>` in `shared/src/business/useCases`.

The tests are spread out, because each permission check is directly associated with the action it's checking prior to executing it. If you have a specific example, we can walk you through the process all the way from the Serverless config of a given stack down to the permission being checked.

Reply:
This is great explanation, thanks!